.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
```

Comment:
I'm curious to see how well this automatic update works. Updating go modules sometimes involves other code changes that come with the new modules.

Reply:
@travisn  Yes in the general development scenario developer will update dependencies but assuming in other scenarios there are updates in upstream dependencies, in those cases this automation will help.